components/Navigation/style.css

```diff
@@ -0,0 +1,34 @@
+.header_menu{
+  display: flex;
+  align-items: center;
+  flex-direction: row !important;
```

Comment:
Using !important is not a good practice as it will .Use can make this class more specific by increasing its specificity